data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
+			`	Elliot introduces you to his family, who thank you for bringing Elliot home. Victor, who was only seven years old when Elliot left and is now twelve, asks you question after question about being a pilot, sometimes asking new questions before you can even answer the old one.`
+			`	"Are you going back to the spaceport, <first>?" the taxi driver yells from the driveway. "I need to get back to the city as soon as I can."
+			`	You say goodbye to Elliot and to his family. "Thanks again, <first>," Elliot says. "I'll always owe you."`
+
+
+
+mission "Unwanted Cargo Trigger"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		random < 10
+		"cargo space" > 0
+	on offer
+		event "hai in cargo" 14
+		fail
+
+event "hai in cargo"
+
+mission "Unwanted Cargo"
+	landing
+	description "Bring a Hai child who hid in your cargo hold back to <destination>."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	destination "Allhome"
+	to offer
+		has "event: hai in cargo"
+	on offer
+		conversation
+			`The "Check Cargo" light for your ship begins blinking when you land, meaning that a cargo crate may have come loose in flight. You shut off the ship's engines and walk to the cargo hold.`
+			`	You check every cargo crate in the hold, but can't find anything wrong. Right as you decide to leave, one of the smaller crates begins rocking back and forth.`
+			branch translator
+				has "Human Cultural Archives: done"
+				
+			`	You hear a voice come from the crate, but it is not a language you understand. The voice continues to speak, and you recognize it as the Hai language.`
+			choice
+				`	(Open the crate.)`
+			`	You open the crate and are met face to face with a Hai child. You aren't sure how old it is, but it looks to be about the height of a one year old human child. It tries to communicate with you, but begins to cry when it sees the confused look on your face.`
+			choice
+				`	"Don't cry, little guy."`
+					goto cry
+				`	(Pick up the Hai.)`
+				
+			`	The child waves its hands around when you try to pick it up, making it impossible to grab on to. After waiting for a few minutes, the child finally calms down, eventually going to sleep in the crate.`
+				goto end
+			
+			label cry
+			`	The child begins to cry even louder than before. After waiting for a few minutes, the child finally calms down, eventually going to sleep in the crate.`
+				goto end
+			
+			label translator
+			`	You hear a voice come from the crate, and surprisingly your Hai translator begins speaking. "Let me out! I want to go home!"`
+			choice
+				`	(Open the crate.)`
+			`	You open the crate and are met eye to eye with a Hai child. You aren't sure how old it is, but it looks to be about the height of a one year old human child. "I want my mommy!" it says as its eyes begin to tear up.`
```

Comment:
Having two small children of my own, I can tell you that one doesn't generally measure one year-olds by height; many of them can't stand up yet. A more appropriate word would be "size".